lang/it/texts/nutriscore.html

```diff
@@ -1,7 +1,7 @@
 <!-- no side column -->
 
 <styles>
-@media only screen and (max-width: 40em) { h2 { font-weight:bold; font-size:1.2em;} }
+@media solo schermo e (larghezza massima: 40em) {h2 {font-weight: bold; font-size: 1.2em;}}
```

Comment:
This shouldn't be translatable and must not be translated.
```suggestion
@media only screen and (max-width: 40em) { h2 { font-weight:bold; font-size:1.2em;} }
```

Reply:
I moved the CSS styles out of the text file (the English one, hopefully Crowdin will then remove it from all the other ones) into the scss